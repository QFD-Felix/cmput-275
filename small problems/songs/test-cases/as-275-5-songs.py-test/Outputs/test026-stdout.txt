14 79 10 3 24 3 10 25 3 29 11 11 1 4 1 9 3 8 0 30 13 2 7 3 16 2 4 14 47 8 1 31 11 12 1 4 37 10 16 38 17 5 6 7 8 6 9 23 19 4 33 15 41 5 15 30 0 2 3 5 0 20 3 0 27 62 29 7 0 20 6 0 0 11 5 7 18 1 9 21 6 20 8 16 14 7 22 10 0 45 23 16 32 13 8 15 0 8 21 43 4 29 16 0 11 9 3 28 33 7 1 1 8 4 11 11 5 6 25 0 28 12 0 5 7 4 6 10 10 32 15 9 22 23 0 0 0 10 8 0 18 0 2 6 11 5 0 0 4 0 1 9 0 0 0 2 0 2 0 0 0 0 0
