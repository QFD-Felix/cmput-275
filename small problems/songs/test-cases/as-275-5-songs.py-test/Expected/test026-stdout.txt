14 85 10 3 11 3 12 27 3 7 13 13 1 6 3 11 14 23 3 8 0 30 13 2 7 3 16 2 4 14 51 8 1 31 11 12 1 4 41 10 16 42 17 5 6 7 8 6 9 23 19 4 22 15 45 5 15 34 0 2 3 5 0 24 3 0 31 67 33 11 0 2 10 4 21 0 0 12 0 11 5 7 18 1 9 21 6 20 8 16 14 7 22 10 0 46 23 16 33 13 8 15 0 8 21 45 4 30 16 0 11 9 3 29 35 7 1 1 8 4 12 9 5 6 24 0 30 13 0 6 8 2 4 6 10 10 33 16 9 23 24 0 0 0 11 9 0 19 0 2 2 6 11 5 0 0 4 0 1 9 0 0 0 2 0 2 0 0 0 0 0
